test/framework/integration_framework/test_irohad.hpp

```diff
@@ -81,6 +81,10 @@ namespace integration_framework {
       return status_bus_;
     }
 
+    const auto &getStorage() {
+      return storage;
+    }
```

Comment:
Since you have been touching that file, could you please update license header too?